docs/cookbooks/multus/README.md

```diff
@@ -0,0 +1,339 @@
+# Using Antrea with Multus
+
+This guide will describe how to use Project Antrea with
+[Multus](https://github.com/intel/multus-cni), in order to attach multiple
+network interfaces to Pods. In this scenario, Antrea is used for the default
+network, i.e. it is the CNI plugin which provisions the "primary" network
+interface ("eth0") for each Pod. For the sake of this guide, we will use the
+[macvlan](https://github.com/containernetworking/plugins/tree/master/plugins/main/macvlan)
+CNI plugin to provision secondary network interfaces for selected Pods, but
+similar steps should apply for other plugins as well,
+e.g. [ipvlan](https://github.com/containernetworking/plugins/tree/master/plugins/main/ipvlan).
+
+## Prerequisites
+
+The only prerequisites are:
+
+ * a K8s cluster (Linux Nodes) running a K8s version supported by Antrea. At the
+   time of writing, we recommend version 1.16 or later. Typically the cluster
+   needs to be running on a network infrastructure that you control. For
+   example, using macvlan networking will not work on public clouds like AWS.
+ * [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+All the required software will be deployed using YAML manifests, and the
+corresponding container images will be downloaded from public registries.
+
+For the sake of this guide, we will use macvlan in "bridge" mode, which supports
+the creation of multiple subinterfaces on one parent interface, and connects
+them all using a bridge. For more information on the different macvlan modes,
+you can refer to this [blog post](https://hicu.be/bridge-vs-macvlan). Macvlan in
+"bridge" mode requires the network to be able to handle "promiscuous mode", as
+the same physical interface / virtual adapter ends up being assigned multiple
+MAC addresses. When using a virtual network for the Nodes, some configuration
+changes are usually required, which depend on the virtualization technology. For
+example:
+
+ * when using VirtualBox and [Internal
+   Networking](https://www.virtualbox.org/manual/ch06.html#network_internal),
+   set the `Promiscuous Mode` to `Allow All`
+ * when using VMware Fusion, enable "promiscuous mode" in the guest (Node) for
+   the appropriate interface (e.g. using `ifconfig`); this may prompt for your
+   password on the host unless you uncheck `Require authentication to enter
+   promiscuous mode` in the Network Preferences
```

Comment:
Network Preferences -> `Network Preferences`?

Reply:
Replaced with the actual menu names: `Preferences ... > Network`